lib/Config.js

```diff
@@ -9,6 +9,25 @@ import authDataChecker from './auth/in_memory/checker';
 const defaultHealthChecks = { allowFrom: ['127.0.0.1/8', '::1'] };
 
 const defaultLocalCache = { host: '127.0.0.1', port: 6379 };
+
+function sproxydAssert(configSproxyd, cb) {
+    if (configSproxyd.bootstrap !== undefined) {
+        assert(Array.isArray(configSproxyd.bootstrap)
+            && configSproxyd.bootstrap
+                .every(e => typeof e === 'string'),
+            'bad config: sproxyd.bootstrap must be a list of strings');
+        assert(configSproxyd.bootstrap.length > 0,
+                'sproxyd bootstrap list is empty');
```

Comment:
Perhaps add "bad config:" here as well and use property syntax as in the other assertion statements.